src/fake/Roc.ts

```diff
@@ -64,14 +67,35 @@ export class FakeDocument extends BaseRocDocument<FakeRoc> {
 
     this.roc.data[this.uuid].push(newDocument);
     this.value = newDocument;
-    return this;
+    return newDocument;
+  }
+
+  public async addGroups(groups: string | string[]) {
```

Comment:
Can we automatically dedupe the owners array? Meaning not add groups that are already in it